docs/sources/tempo/configuration/sidecar-proxy.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Configure sidecar proxies
+menuTitle: Configure sidecar proxies
+description: Configure Tempo Distributed to run with sidecar proxies
+weight: 36
+---
+
+# Run Tempo Distributed with sidecar proxies
+
+You can run inter-pod gRPC traffic run through a sidecar proxy. These proxies can be useful when using a service proxy, like Envoy, or a service mesh, like Istio. 
```

Comment:
```suggestion
You can route inter-pod gRPC traffic run through a sidecar proxy to meet requirements such as custom security, routing, or logging.  Common examples include Envoy, Nginx, Traefik, or service meshes like Istio and Linkerd. 
```

Maybe something like this.

Reply:
This is exactly what we needed. Thank you.